src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+**Spans**
 
-[{% asset performance/anatomy-of-transaction.png alt="Diagram illustrating how a transaction is composed of many spans." %}]({% asset performance/anatomy-of-transaction.png @path %})
+The majority of the data in a transaction resides in the individual spans the transaction contains. Span data includes:
 
-Traces in Sentry are segmented into [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) called [**transactions**]({%- link _documentation/performance/performance-glossary.md -%}#transaction). When instrumenting the application with tracing, collected spans are grouped within an encompassing top-level span called the **transaction span**. This notion of a transaction is specific to Sentry.
+- `parent_span_id`: ties the span to its parent span
+- `op`: short string identifying the type of operation the span is measuring
+- `start_timestamp`: when the span was opened
+- `end_timestamp`: when the span was closed
+- `description`: longer description of the span's operation, often specific to that instance (optional)
+- `status`: short code indicating operation's status (optional)
+- `tags`: key-value pairs holding additional data about the span (optional)
+- `data`: arbitrarily-structured additional data about the span (optional)
 
-If you are collecting transactions from more than a single machine, you will likely encounter [**clock skew**]({%- link _documentation/performance/performance-glossary.md -%}#clock-skew).
+An example use of the `op` and `description` properties together is `op: sql.query` and `description: SELECT * FROM users WHERE last_active < DATE_SUB(CURRENT_DATE, INTERVAL 1 YEAR)`. The `status` property is often used to indicate the success or failure of the span's operation, or for a response code in the case of HTTP requests. Finally, `tags` and `data` allow you attach further contextual information to the span, such as `function: middleware.auth.is_authenticated` for a function call or `request: {url: ..., headers: ... , body: ...}` for an HTTP request.
+  
+### Further Information
 
-## Sampling Transactions
+A few more important points about traces, transactions, and spans, and the way they relate to one another:
 
-**We strongly recommend sampling your transactions.**
+**Trace Duration**
 
-When you enable sampling for transaction events in Sentry, you choose a percentage of collected transactions to send to Sentry. For example, if you had an endpoint that received 1000 requests per minute, a sampling rate of 0.25 would result in 250 transactions (25%) being sent to Sentry.
+Because a trace just is a collection of transactions, traces don't have their own start and end times. Instead, a trace begins when its earliest transaction starts, and ends when its latest transaction ends. As a result, you can't explicitly start or end a trace directly. Instead, you create a trace by creating the first transaction in that trace, and you complete a trace by completing all of transactions it contains.
```

Comment:
Does it make sense to reverse?

```suggestion
Because a trace is just a collection of transactions, traces don't have their own start and end times. Instead, a trace begins when its earliest transaction starts, and ends when its latest transaction ends. As a result, you can't explicitly start or end a trace directly. Instead, you create a trace by creating the first transaction in that trace, and you complete a trace by completing all of transactions it contains.
```

Reply:
🤔

Not to my ear.

The difference is subtle, and thinking about it is harder because "to be" is the exception to every rule (except when it isn't, or when it sometimes is and sometimes isn't). But if we take some other examples...

"I just can't make up my mind" (no matter how hard I try) vs "I can't just make up my mind" (I also have to act on it)

"He just vacuumed the hall" (so please take off your shoes) vs "he vacuumed just the hall" (even though he was supposed to do the bedrooms, too)

"She just took the cookie" (and didn't even say thanks) vs "she took just the cookie" (and left the brownie for me)

"The cow just ate grass" (and didn't do anything else all day) vs "the cow ate just grass" (even though most cows also like... actually, wait, what do cows like?)

"Just stand there" (don't move) vs "stand just there" (not a foot to the left or the right)

Anyway, you get the point. When the "just" comes before the verb, it modifies the verb, whether to emphasize it ("I just can't" = "I really really can't") or to restrict it ("just stand" = "do nothing but stand") or to place it in time ("just vacuumed" = "finished vacuuming very recently").

When it comes after, it modifies whatever else is in the sentence, whether to restrict it "just grass" = "nothing else besides grass") or to make it more precise ("just there" = "exactly there").

Circling back around... "is" is weird, and so only some of the meanings of "just" could even apply to it. The fact that none of those meanings really make sense here is I think why "just is a collection" sounds strange to me. "Just a collection," on the other hand, is no different than any of the other examples above.

(And now I will stop grammar-nerd-ing at you, I promise! 😛 )